go-controller/pkg/util/gateway-init.go

```diff
@@ -182,18 +184,18 @@ func GatewayInit(clusterIPSubnet []*net.IPNet, hostSubnet *net.IPNet, joinSubnet
 	stdout, stderr, err = RunOVNNbctl("set", "logical_router",
 		gatewayRouter, "options:lb_force_snat_ip="+gwLRPIp.String())
 	if err != nil {
-		return fmt.Errorf("Failed to set logical router, stdout: %q, "+
-			"stderr: %q, error: %v", stdout, stderr, err)
+		return fmt.Errorf("Failed to set logical router %s, stdout: %q, "+
+			"stderr: %q, error: %v", gatewayRouter, stdout, stderr, err)
```

Comment:
can we add more context to say what we are setting

return fmt.Errorf("failed to set logical router %s's lb_force_snat_ip option, stdout: %q, "+
			"stderr: %q, error: %v", gatewayRouter, stdout, stderr, err)